src/wasm/wasm-binary.cpp

```diff
@@ -636,8 +636,13 @@ void WasmBinaryWriter::visitSetLocal(SetLocal *curr) {
 }
 
 void WasmBinaryWriter::visitGetGlobal(GetGlobal *curr) {
-  if (debug) std::cerr << "zz node: GetGlobal " << (o.size() + 1) << std::endl;
-  o << int8_t(BinaryConsts::GetGlobal) << U32LEB(getGlobalIndex(curr->name));
+  if (curr->name.is()) {
```

Comment:
I don't think we should ever call `visitGetGlobal` in a module without globals, so I can't see any reason why `curr` would not have a name. How is this happening?